### ISSUE TITLE
font.layout hangs forever

### ISSUE THREAD

**Issue by pzavolinsky** [devongovett/fontkit#123]
`font.layout` when applied to a (weird) _slash-looking_ char (`'⁄'`) hangs forever.

----

Scripted repro:

```bash
mkdir fk-bug
cd fk-bug
echo '{}' > package.json

npm i node-fetch fontkit

echo "const fontkit = require('fontkit');
const fetch = require('node-fetch');
fetch('https://app.mural.ly/static/fonts/proximanova-bold-webfont.ttf')
.then(resp => resp.buffer())
.then(buf =>
  console.log('trying to compute layout')
  || console.log('worked!', fontkit.create(buf).layout('⁄'))
);" > bug.js

node bug.js
```

----

Manual repro:
`package.json`:

```json
{
  "dependencies": {
    "fontkit": "^1.7.6",
    "node-fetch": "^1.7.1"
  }
}
```

`bug.js`:

```js
const fontkit = require('fontkit');
const fetch = require('node-fetch');
fetch('https://app.mural.ly/static/fonts/proximanova-bold-webfont.ttf')
.then(resp => resp.buffer())
.then(buf =>
  console.log('trying to compute layout')
  || console.log('worked!', fontkit.create(buf).layout('⁄'))
);
```

Expected result
----------------

You should see the `worked!` message

Actual result
-------------

`font.layout` just hangs.

Notes
------

The text we pass to layout looks like a slash but it is NOT a slash:

```js
('⁄').codePointAt(0) // returns  8260
```

**Comment by Pomax** [comment]
On a technical elaboration note, rather than a "weird slash-looking char" this is `U+2044 FRACTION SLASH : solidus (in typography)`

**Comment by khaledhosny** [comment]
That is a fraction slash, looks like the code applying automatic fractions is entering into infinite loop.

**Comment by Pomax** [comment]
@pzavolinsky does this happen for any font you've tried, or only some fonts?

**Comment by pzavolinsky** [comment]
Honestly I've only tried with the one in the repro (`proxima-nova`).

**Comment by pzavolinsky** [comment]
Here is the complete text I was measuring originally:

> What are we doing in the next  X ⁄ Y ⁄ Z  months?

**Comment by khaledhosny** [comment]
Does not seem to be font-dependent. I have a patch that fixes this but causes a test failure, debugging…

**Comment by khaledhosny** [comment]
That string above is bogus though, fraction slash is for fractions not dates.

**Comment by pzavolinsky** [comment]
@khaledhosny point taken, unfortunately, a customer of ours typed that string into a textbox in our app (most likely a copy/pasted from some other app).

**Comment by pzavolinsky** [comment]
> Does not seem to be font-dependent.
Agreed, just tried this [Arial font](https://github.com/JotJunior/PHP-Boleto-ZF2/blob/master/public/assets/fonts/arial.ttf) and it fails as well.

**Comment by khaledhosny** [comment]
Should be fixed in https://github.com/devongovett/fontkit/pull/124.

**Comment by pzavolinsky** [comment]
Excellent, thanks @khaledhosny !

@devongovett I saw you merged #124 to master already. Any chance you could push `fontkit@1.7.7` with that fix?

**Comment by devongovett** [comment]
Done
